fsharp-backend/src/LibExecution/StdLib/LibHttpClient.fs

```diff
@@ -19,109 +19,98 @@ let incorrectArgs = LibExecution.Errors.incorrectArgs
-//             let auth_string =
-//               Unicode_string.append
-//                 (Unicode_string.of_string_exn "Bearer ")
-//                 token
-//             in
-//             DObj (Map.singleton "Authorization" (DStr auth_string))
-//         | _ ->
-//             incorrectArgs ())
-//   ; sqlSpec = NotYetImplementedTODO
-//   ; previewable = Pure
-//   ; deprecated = NotDeprecated } ]
-//
+let fns : List<BuiltInFn> =
+  [ { name = fn "HttpClient" "formContentType" 0
+      parameters = []
+      returnType = TDict varA
+      description =
+        "Returns an object with 'Content-Type' for url-encoded HTML forms"
+      fn =
+        (function
+        | _, [] ->
+            Value(
+              DObj(
+                Map.ofList [ "Content-Type", DStr "application/x-www-form-urlencoded" ]
+              )
+            )
+        | _ -> incorrectArgs ())
+      sqlSpec = NotYetImplementedTODO
+      previewable = Pure
+      deprecated = NotDeprecated }
+    { name = fn "HttpClient" "jsonContentType" 0
+      parameters = []
+      returnType = TDict varA
+      description = "Returns an object with 'Content-Type' for JSON"
+      fn =
+        (function
+        | _, [] ->
+            Value(
+              DObj(
+                Map.ofList [ "Content-Type", DStr "application/json; charset=utf-8" ]
+              )
+            )
+        | _ -> incorrectArgs ())
+      sqlSpec = NotYetImplementedTODO
+      previewable = Pure
+      deprecated = NotDeprecated }
+    { name = fn "HttpClient" "plainTextContentType" 0
+      parameters = []
+      returnType = TDict varA
+      description = "Returns an object with 'Content-Type' for plain text"
+      fn =
+        (function
+        | _, [] ->
+            Value(
+              DObj(Map.ofList [ "Content-Type", DStr "text/plain; charset=utf-8" ])
+            )
+        | _ -> incorrectArgs ())
+      sqlSpec = NotYetImplementedTODO
+      previewable = Pure
+      deprecated = NotDeprecated }
+    { name = fn "HttpClient" "htmlContentType" 0
+      parameters = []
+      returnType = TDict varA
+      description = "Returns an object with 'Content-Type' for html"
+      fn =
+        (function
+        | _, [] ->
+            Value(
+              DObj(Map.ofList [ "Content-Type", DStr "text/html; charset=utf-8" ])
+            )
+        | _ -> incorrectArgs ())
+      sqlSpec = NotYetImplementedTODO
+      previewable = Pure
+      deprecated = NotDeprecated }
+    { name = fn "HttpClient" "bearerToken" 0
+      parameters = [ Param.make "token" TStr "" ]
+      returnType = TDict varA
+      description = "Returns an object with 'Authorization' set to the passed token"
+      fn =
+        (function
+        | _, [ DStr token ] ->
+            let authString =
+              System.Text.Encoding.UTF8.GetString(
+                Array.append
+                  (System.Text.Encoding.UTF8.GetBytes "Bearer ")
+                  (System.Text.Encoding.UTF8.GetBytes token)
```

Comment:
I think you should use string append here instead of converting to bytes.

Reply:
Thank you! I will add the changes